packages/components/src/components/checkbox-group/checkbox-group.stories.ts

```diff
@@ -40,102 +43,66 @@ export const Default = {
-              }
-            ]
-          }
-        ],
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+      <sd-checkbox-group size="sm">
+        <sd-checkbox value="1">Option 1</sd-checkbox>
+        <sd-checkbox value="2">Option 2</sd-checkbox>
+        <sd-checkbox value="3">Option 3</sd-checkbox>
+      </sd-checkbox-group>
+    </div>
+  `
 };
 
 /**
- * Use the `form-control`, `form-control-label` and `form-control-input` part selectors to customize the checkbox-group.
+ * Use the `label`attribute to add a label on top.
  */
-export const Parts = {
-  parameters: {
-    controls: { exclude: ['form-control', 'form-control-label', 'form-control-input'] }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'template',
-          name: 'sd-checkbox-group::part(...){outline: solid 2px red}',
-          values: ['form-control', 'form-control-label', 'form-control-input'].map(part => {
-            return {
-              title: part,
-              value: `<style>#part-${part} sd-checkbox-group::part(${part}){outline: solid 2px red}</style><div id="part-${part}">%TEMPLATE%</div>`
-            };
-          })
-        }
-      },
-      constants: [{ type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' }],
-      args
-    });
-  }
+
+export const Label = {
+  name: 'Label',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-checkbox-group orientation="vertical">
+        <label slot="label">Vertical Group</label>
+        <sd-checkbox value="1">Option 1</sd-checkbox>
+        <sd-checkbox value="2">Option 2</sd-checkbox>
+        <sd-checkbox value="3">Option 3</sd-checkbox>
+      </sd-checkbox-group>
+    </div>
+  `
 };
 
 /**
- * sd-checkbox-group is fully accessibile via keyboard.
+ * Use the `orientation` attribute to change the layout of the checkboxes.
  */
-export const Mouseless = {
-  render: (args: any) => {
-    return html`<div class="mouseless">${generateTemplate({ args })}</div>`;
-  },
-
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-checkbox-group');
-    await waitUntil(() => el?.shadowRoot?.querySelector('label'));
 
-    if (el?.shadowRoot) {
-      const label = el.shadowRoot.querySelector('label');
-      if (label) {
-        await userEvent.type(label, '{space}', { pointerEventsCheck: 0 });
-      }
-    }
-  }
+export const Orientation = {
+  name: 'Orientation',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-checkbox-group orientation="vertical">
+        <label slot="label">Vertical Group</label>
+        <sd-checkbox value="1">Option 1</sd-checkbox>
+        <sd-checkbox value="2">Option 2</sd-checkbox>
+        <sd-checkbox value="3">Option 3</sd-checkbox>
+      </sd-checkbox-group>
+      <sd-checkbox-group orientation="horizontal">
```

Comment:
Hide group label

Reply:
Updated

Reply:
@smfonseca in the Radio Group you should do the same please.